async-nats/src/jetstream/mod.rs

```diff
@@ -11,11 +11,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::Client;
+use crate::{Client, Message};
 
 pub mod consumer;
 pub mod context;
 pub mod publish;
+pub mod pull;
```

Comment:
Move into consumer module?

```suggestion
pub mod pull;
```

Reply:
That would require move consuer to sepearte directory with its own mod.rs. 
Was about to oppose that, but that makes sense structure wise:

core nats
-- jetstream
---- consumers
---- kv
---- object store

agree?

Reply:
yeah, I went with consumer :)